fastfood/food.py

```diff
@@ -42,33 +41,23 @@ def build_cookbook(build_config, templatepack, cookbooks_home,
     with open(build_config) as cfg:
         cfg = json.load(cfg)
 
+    cookbook_name = cfg['name']
     cookbook = None
-    if cookbook_path:
-        try:
-            cookbook = book.CookBook(cookbook_path)
-        # change this to a more specific error
-        except ValueError:
-            # create a new one please
-            pass
+    cookbook_path = os.path.join(cookbooks_home, cookbook_name)
 
     written_files = []
     if not cookbook:
```

Comment:
Yes, that's right. And create_new_cookbook now is only used to ensure the cookbook directory exists. It returns, and then stencil processing begins.